cfpb/cfgov-refresh
# Moving /servicemember into cfgov-refresh

Moving servicemember into cfgov-refresh


## Changes

Created Jinja template and modified css code to move /servicemember into cfgov-refresh. I had to modify the non-responsive header / footer css to use the new cf4 atomic styles.
 

## Testing

1. Visit `http://localhost:8000/servicemembers/` and ensure the page renders as expected.
2. Click the pdf download links and ensure the pdfs render as expected.

## Screenshots

- Didn't add a flag because of the Apache entry.


## Checklist
